resources/tools.json

```diff
@@ -747,6 +747,19 @@
         }
       },
       "test": "simple-phpunit --version"
+    },
+    {
+      "name": "larastan",
+      "summary": "PHPStan extension for Laravel",
+      "website": "https://github.com/nunomaduro/larastan",
+      "command": {
+        "composer-bin-plugin": {
+          "package": "nunomaduro/larastan",
+          "namespace": "phpstan"
+        }
+      },
+      "test": "true",
```

Comment:
can you verify the plugin is install here please? 

`composer global show nunomaduro/larastan`

Reply:
@jakzal it has broken the build test. I'm not sure what happened...in the logs file I can see the plugin being installed, but `composer show` fails

Reply:
Just realised I made a mistake above. The command should be:

```composer global bin phpstan show ergebnis/phpstan-rules```

I also noticed that many phpstan plugins use `true`. I'll fix that in a separate PR.

Reply:
Can you remove the tags section? We don't support PHP 7.1 at all.